ngrinder-core/src/main/java/net/grinder/SingleConsole.java

```diff
@@ -349,6 +359,26 @@ public String getConsoleIP() {
 		return this.getConsoleProperties().getConsoleHost();
 	}
 
+	@Override
+	public void onAcceptMd5Listener(Set<String> md5) {
+		this.agentCachedDistFilesMd5List.add(md5);
+	}
+
+	/**
+	 * Send Md5 of unnecessary files to agents for refresh agent's distribution cache directory.
+	 *
+	 * @param distFilesMd5 Required file's md5 for currently running test
+	 */
+	public void sendDistFilesMd5ToAgents(Set<String> distFilesMd5) {
+		stream(processReports)
+			.parallel()

```

Comment:
I don't want to use parallelStream.
https://multifrontgarden.tistory.com/254

Reply:
Using parallel streams elsewhere without a separate thread pool might be a problem.
so modified to work in separate thread pool.

Reply:
Using ForkJoinPool is good :)
If the two performance differences aren't big, I don't want them in parallel.

Can you tell me the difference on average based on five agents?

Reply:
We are already sending agent start messages in parallel.
So, I used parallel in the same context.

There seems to be no big difference in performance.